### PR TITLE
EDUCATOR-5433: clear validation errors on select change

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Clear "File Types Whitelist" errors when the "File Upload Types" select changes

**What changed?**

- I don't understand what the original thing was doing... I mean, I know what it was literally doing, but I couldn't find `is-shown` anywhere in CSS files or anything. I don't know why that was being set. It seemed to do nothing, and even when calling clearValidationErrors, it still showed up. That was the main change here.

**Testing Instructions**
- go to studio, into an ORA, and into the ORA settings
- enable file upload responses
- "File Upload Types" -> Custom
- clear the box. hit save, there should be an error on the field
- Change File Upload Types to another option. The errors should no longer appear

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [EDUCATOR-5433](https://openedx.atlassian.net/browse/EDUCATOR-5433)

FIY: @edx/masters-devs-gta

### CODE REVIEW

**Comment by jansenk** [comment]
.............. why is it saying edx-requirement-bot is an author on this

**Review by muselesscreator — COMMENTED** [review]
LOL!  This actually makes sense to me.  (Have not tested)

**Review by nsprenkle — COMMENTED** [review]
LGTM, but I also haven't tested locally. Will test locally when I'm done fighting w/ devstack

**Review by nsprenkle — APPROVED** [review]
Tested locally, looks good :shipit:
